.buildkite/docker-build-push-explorerApi.nix

```
@@ -76,6 +71,8 @@ in
         echo "Pushing $fullrepo:$tag"
         docker push "$fullrepo:$tag"
       fi
+      echo "Cleaning up with docker system prune"
+      docker system prune -a -f
```

```suggestion
      docker system prune -f
```
Only required if containers are started, so this is superfluous